tests/dast/Dockerfile

```diff
@@ -36,4 +36,5 @@ RUN mkdir -p /.local
 RUN useradd podman
 RUN chown podman:podman -R /.local
 RUN chmod 777 /.local
+RUN podman unshare chown 1000 results
```

Comment:
I think it's better to squash `chown` and `chmod` into a single RUN command, cause each RUN is creating another layer in the image.

Reply:
not sure how that will work with &&

Reply:
This is how you combine several similar commands into one command, the difference is in the image layers.
```
RUN chown podman:podman -R /.local
    chmod 777 /.local
```

You can even include more of the RUN commands that you are using, if they share the same logic layer.